ui/app/AppLayouts/Profile/popups/RenameAccontModal.qml

```diff
@@ -64,8 +64,10 @@ StatusModal {
 
             onIconClicked: {
                 popup.emojiPopup.open()
-                popup.emojiPopup.x = popup.x + accountNameInput.x + Theme.padding
-                popup.emojiPopup.y = popup.y + contentItem.y + accountNameInput.y + accountNameInput.height +  Theme.halfPadding
+                if (!popup.emojiPopup.bottomSheet) {
```

Comment:
I noticed that without that condition the popup doesn't open in the non-bottomSheet mode. That's quite strange because for that mode that condition should not matter at all :thinking: Maybe there are some initialization-order intricacies

Reply:
works now dont know why but it does :D

https://github.com/user-attachments/assets/4af30049-7bae-44d4-9617-2c14ddf9a347